lib/cocoapods/sandbox.rb

```diff
@@ -150,7 +163,7 @@ def target_support_files_dir(name)
     #
     # @return [Pathname] the path of the Pod.
     #
-    def pod_dir(name)
+    def pod_realdir(name)
```

Comment:
this seems weird to me, why do we need two methods for the `pod_dir` ?

Reply:
would calling `realpath` on the result of `pod_dir` be different from this implementation?

Reply:
I create a new `pod_dir` method, and is always `${PODS_ROOT}/PODNAME`. It is symbol link path when the pod is local pod ( with `:path` ).
So this `pod_realdir` will return a realpath of the pod.